tools/kind-with-registry.sh

```diff
@@ -41,7 +41,7 @@ esac
 running="$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)"
 if [ "${running}" != 'true' ]; then
   docker run \
-    -d --restart=always -p "${reg_port}:5000" --name "${reg_name}" \
```

Comment:
I’m assuming, based on what you’ve said outside this PR, that this change is a necessary part of the fix. Do you know (or at least have a working theory) what was going wrong on Macs when the `127.0.0.1` wasn’t in this command?

Reply:
0.0.0.0 is a non routable address while 127.0.0.1 is the right loopback address. The difference is that on Linux docker is native. But on a mac, you are running docker in a microvm.